cloudformation/cloudformation.yaml

```diff
@@ -298,4 +298,56 @@ Resources:
       Properties:
           Description: Deploys fulfilment-api into an environment/stage
           RestApiId: !Ref FulfilmentAPI
-      DependsOn: FulfilmentMethod
\ No newline at end of file
+      DependsOn: FulfilmentMethod
+
+  ScheduledRule:
+    Type: "AWS::Events::Rule"
+    Properties:
+      Description: "TriggerFulfilment"
+      ScheduleExpression: "cron(00 11 ? * mon-fri *)"
```

Comment:
we should confirm is this is the correct time. One thing to consider is that this is gmt  so in the summer this is 12:00 bst